documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -160,12 +160,53 @@ include::{testDir}/example/DisplayNameGeneratorDemo.java[tags=user_guide]
   | | +-- For example, year -1 is not supported. [OK]
   | | '-- For example, year -4 is not supported. [OK]
   | '-- if it is zero() [OK]
-  '-- A year is a leap year... [OK]
-    +-- A year is a leap year if it is divisible by 4 but not by 100. [OK]
-    '-- A year is a leap year if it is one of the following years. [OK]
-      +-- Year 2016 is a leap year. [OK]
-      +-- Year 2020 is a leap year. [OK]
-      '-- Year 2048 is a leap year. [OK]
+  '-- A year is a leap year [OK]
+    +-- A year is a leap year, if it is divisible by 4 but not by 100. [OK]
+    '-- A year is a leap year, if it is one of the following years. [OK]
```

Comment:
We might add the deleted lines (166-168) after this sentence just so that we are actually adding the examples as we state above

Reply:
Sure! no problem re-adding the examples, I think I chopped them up thinking it would be simpler.

Reply:
You are completely right. I just meant, probably that line should also be removed with if we remove the examples. It may be confusing for the reader. It's your take on adding or removing the examples, you are right, probably every reader would know examples of leap year. The comment was to point out a discrepancy. Thanks for looking into it!